src/server/auth/middleware/bearerAuth.ts

```diff
@@ -42,7 +42,7 @@ export function requireBearerAuth({ verifier, requiredScopes = [], resourceMetad
     try {
       const authHeader = req.headers.authorization;
       if (!authHeader) {
-        throw new InvalidTokenError("Missing Authorization header");
+        throw new MissingAuthenticationError("Missing Authorization header");
```

Comment:
```suggestion
        throw new MissingAuthorizationError("Missing Authorization header");
```

here and elsewhere, this should line up with what's missing (specifically Authorization)